src/server/home/controller.js

```diff
@@ -6,7 +8,7 @@
 export const homeController = {
   handler(_request, h) {
     return h.view('home/index', {
-      pageTitle: 'Apply for a Bovine Tuberculosis (TB) movement licence',
+      pageTitle: config.get('serviceName'),
       heading: 'Apply for a Bovine Tuberculosis (TB) movement licence'
```

Comment:
Do we still need this property then?

Reply:
Does the title tag not need to be changed too? @hughfdjackson ?

Reply:
It does - that's the second AC in the ticket related to this PR